torchci/pages/minihud/[repoOwner]/[repoName]/[branch]/[[...page]].tsx

```diff
@@ -38,13 +40,17 @@ function includesCaseInsensitive(value: string, pattern: string): boolean {
   return value.toLowerCase().includes(pattern.toLowerCase());
 }
 
-function FailedJob({ job }: { job: JobData }) {
+function FailedJob({
+  job,
+  classification,
+}: {
+  job: JobData;
```

Comment:
What would it take to have `JobData` include the `classification`?  That would simplify a lot of the code here

Reply:
It would probably require grouping them data into a single table/query since these data types reflect what we receive from the query.

Reply:
Yeah, putting them in a single query would work pretty nicely.  If you're short on time we can save this for a future day though.